frontend/cypress/integration/patients.spec.js

```diff
@@ -1,103 +1,101 @@
     cy.intercept('DELETE', '/api/patients/*')
-      .as('deletePatients');
-    cy.intercept('PUT', '/api/patients/*')
-      .as('modifyPatients');
+      .as('deleteEtudiants');
     cy.intercept('GET', '/api/patients/*')
-      .as('patient');
+      .as('etudiant');
     cy.intercept('GET', '/api/config')
       .as('config');
 
     resetDB();
     loginAsDefault();
 
-    cy.visit('/psychologue/mes-patients');
+    cy.visit('/psychologue/mes-etudiants');
     cy.wait('@config');
-    cy.wait('@patients');
+    cy.wait('@etudiants');
   });
 
   describe('Display', () => {
-    it('should get patients', () => {
-      cy.get('[data-test-id="patient-row"]')
+    it('should get etudiants', () => {
+      cy.get('[data-test-id="etudiant-row"]')
         .should('have.length', 5);
-      cy.get('[data-test-id="patient-row-missing-info"]')
+      cy.get('[data-test-id="etudiant-row-missing-info"]')
         .should('have.length', 1);
-      cy.get('[data-test-id="patient-row-complete-info"]')
+      cy.get('[data-test-id="etudiant-row-complete-info"]')
         .should('have.length', 4);
-      cy.get('[data-test-id="patients-missing-info"]')
+      cy.get('[data-test-id="etudiants-missing-info"]')
         .should('exist');
     });
   });
 
   describe('Update', () => {
-    it('should update existing patient and update completion info', () => {
-      cy.get('[data-test-id="update-patient-button-large"]')
+    it('should update existing etudiant and update completion info', () => {
+      cy.get('[data-test-id="update-etudiant-button-large"]')
         .first()
         .click();
-      cy.wait('@patient');
+      cy.wait('@etudiant');
 
-      cy.get('[data-test-id="patient-first-name-input"] > input')
+      cy.get('[data-test-id="etudiant-first-name-input"] > input')
         .clear()
         .type('Georges');
-      cy.get('[data-test-id="patient-last-name-input"] > input')
+      cy.get('[data-test-id="etudiant-last-name-input"] > input')
         .clear()
         .type('Moustaki');
-      cy.get('[data-test-id="patient-doctor-name-input"] > input')
+      cy.get('[data-test-id="etudiant-doctor-name-input"] > input')
         .type('My doctor');
-      cy.get('[data-test-id="save-patient-button"]')
+      cy.get('[data-test-id="save-etudiant-button"]')
         .click();
 
-      cy.get('[data-test-id="patient-row"]')
+      cy.get('[data-test-id="etudiant-row"]')
         .should('have.length', 5);
-      cy.get('[data-test-id="patient-row-complete-info"]')
+      cy.get('[data-test-id="etudiant-row-complete-info"]')
         .should('have.length', 5);
-      cy.get('[data-test-id="patients-missing-info"]')
+      cy.get('[data-test-id="etudiants-missing-info"]')
         .should('not.exist');
 
       cy.get('[data-test-id="notification-success"] p')
         .should(
           'have.text',
-          'Le patient Georges Moustaki a bien été modifié.',
+          'Le etudiant Georges Moustaki a bien été modifié.',
```

Comment:
```suggestion
          'L'étudiant Georges Moustaki a bien été modifié.',
```
To be fixed on all places.